randovania/games/samus_returns/json_data/Surface - West.txt

```diff
@@ -31,7 +31,19 @@ Extra - asset_id: collision_camera_000
   > Save Station
       Trivial
   > Event - Proteus Ridley
-      Trivial
+      All of the following:
+          # TODO: Add fight requirements
+          Any of the following:
+              # Energy Requirements
+              Combat (Advanced) or Normal Damage ≥ 599
+              Combat (Beginner) and Normal Damage ≥ 399
+              Combat (Intermediate) and Normal Damage ≥ 199
+          # Dealing Damage
+          Plasma Beam
```

Comment:
since this is marked as todo, i suppose you could ignore this, but if its possible to lemon shoot him, i'd lock that behind some combat

Reply:
I imagine it does work with just Power. Ridley is weird because there's a reduction on only Plasma. The fight needs tweaking for proper beam balance anyway.

Reply:
I imagine it does work with just Power. Ridley is weird because there's a reduction on only Plasma. The fight needs tweaking for proper beam balance anyway.